engine/class_modules/sc_death_knight.cpp

```diff
@@ -4998,6 +5007,10 @@ struct frost_strike_strike_t : public death_knight_melee_attack_t
     death_knight_melee_attack_t::execute();
 
     trigger_icecap( execute_state );
+
+    if ( p() -> conduits.unleashed_frenzy->ok() )
```

Comment:
Even though this a single statement block please wrap it in braces to ensure consistency across the entire file.

Reply:
Updated as per suggestion